src/task_controller.js

```diff
@@ -0,0 +1,153 @@
+const type = 'task';
+
+/** A mediator between database and user interface */
+class TaskController {
+  /**
+   * Creates a controller with injected database dependency
+   * @param {object} db - object of Database interface
+   * */
+  constructor(db) {
+    this.db = db;
+  }
+
+  /**
+   * Retrieves all tasks
+   * @return {array}
+   */
+  getAll() {
+    return this.db.getAll(type).map((e) => toTask(e));
```

Comment:
Better to take this out into a separate function.

Reply:
I've met people who are against decomposing such small chunks of code, but I'll do it for you, my charming girl

Reply:
At first I thought so too, but it duplicates four times. Thank you for an interesting info, I'll check it out. I'm grateful that you accept my solution, my dear.